### PR TITLE
Fix run

### PR DESCRIPTION
**Pull request by jotadrilo**
This PR expect the application to work as non-root user so `pm2` is started as `bitnami` user in port 8080.

However, the `run.sh` script should run as `root` as it will read environment variables and semaphore from restricted files

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -23,33 +25,46 @@ stop () {
...
+      chown -R bitnami:bitnami ${APP_FOLDER}
+      chown -R bitnami:bitnami ${DATA_FOLDER}
+      chown root:root ${APP_FOLDER}/run.sh
+
+      # Install node modules
+      su bitnami npm install
+
       # Touch semaphore
-      touch .initialized
+      su bitnami ${DATA_FOLDER}/.initialized
```
Comment (tompizmor) [inline run.sh]: no touching no touching??

Comment (jotadrilo) [reply run.sh]: good catch!

**Comment by tompizmor** [comment]
LGTM!!

**Comment by dbarranco** [comment]
lgtm

**Pull request merged**